cluster-state-service/canary/pkg/logger/logger.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2016-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I could. I was trying to keep the canary as independent as possible from the service code. In case we want to move it out.